packages/docregistry/src/registry.ts

```diff
@@ -697,11 +706,11 @@ namespace DocumentRegistry {
     readonly localPath: string;
 
     /**
-     * The current contents model associated with the document
+     * The document metadata, stored as a services contents model.
      *
      * #### Notes
-     * The contents model will be null until the context is ready.
-     * It will have an  empty `contents` field.
+     * This will be null until the context is 'ready'. Since we only store
+     * metadata here, the `.contents` attribute will always be empty.
```

Comment:
I don't think this note is strictly true: on `revert()`, don't we get the full contents model? Also, I ran into a bug the other day where `.format` was also empty. According to the documentation, `contents`, `mimetype`, and `format` are the ones that are empty when we don't get the full model: http://jupyter-notebook.readthedocs.io/en/stable/extending/contents.html#modelcontent

Reply:
What are you saying is the correct explanation that should be there?

Reply:
Just that we should document the actual behavior somewhere. It was the reason for #4602, and is not immediately obvious. I can also flesh it out more in that PR